src/css/components/menu.less

```diff
@@ -0,0 +1,43 @@
+[role="menu"] {
```

Comment:
What is the advantage of using `role instead of a good old class name by the way?